src/clusterfuzz/_internal/metrics/monitor.py

```diff
@@ -51,6 +51,12 @@
 MAXIMUM_DELAY_SECONDS = 2 * 60  # 2 minutes.
 MAX_TIME_SERIES_PER_CALL = 200
 
+# Since `monitoring_v3` is only conditionally imported, pylint complains about
+# any accesses to its members. Define type aliases here once and for all, for
+# use below.
+_TimeInterval = monitoring_v3.types.TimeInterval  # pylint: disable=no-member
```

Comment:
This will cause an exception in the case that `monitoring_v3 = None` above right?

Reply:
I reverted this specific change in https://github.com/google/clusterfuzz/pull/4125/commits/ade6f127bce74b9b932c0ec3c119bba140b14107 for now.